sandbox/package.json

```diff
@@ -10,7 +10,7 @@
     "@types/node": "^16.11.49",
     "@types/react": "^18.0.17",
     "@types/react-dom": "^18.0.6",
-    "configcat-react": "^4.1.0",
+    "configcat-react": "^4.6.0",
```

Comment:
We should set the upcoming version here, otherwise the new features won't work.

Reply:
True that! After the merge I will update it with release PR. Ok?